frontend/src/Socket.js

```diff
@@ -2,6 +2,8 @@ import socketIOClient from 'socket.io-client';
 
 /* eslint-disable new-cap */
 const socket = new socketIOClient('mafia-hands-backend-cd.herokuapp.com');
```

Comment:
Could you add a comment saying: 

"Uncomment if you would like to use the deployed server"